start-site/src/main/resources/application.yml

```diff
@@ -86,19 +86,19 @@ initializr:
         mappings:
           - compatibilityRange: "[2.2.0.RELEASE,2.4.0.M1)"
             version: Hoxton.SR12
-          - compatibilityRange: "[2.4.0.M1,2.4.0.M1]"
-            version: 2020.0.0-M3
-            repositories: spring-milestones
-          - compatibilityRange: "[2.4.0.M2,2.4.0-M3]"
-            version: 2020.0.0-M4
-            repositories: spring-milestones
```

Comment:
We can prune older metadata but I am not keen to do this as par of adding `2021.0.0-M1`.

Reply:
OK should I submit a separate PR to remove these then?

Reply:
No need to, I can take care of that.

Reply:
Thanks!